### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by kelvinlau20100**

### CODE REVIEW

File: zeta-cpi/package.json [hunk 1]
```diff
@@ -1,16 +1,16 @@
 {
   "dependencies": {
-    "@project-serum/anchor": "^0.18.2",
+    "@project-serum/anchor": "^0.24.2",
     "@solana/spl-token": "^0.1.8",
-    "@zetamarkets/sdk": "^0.9.4",
-    "dotenv": "^10.0.0",
-    "node-fetch": "^3.1.0"
+    "@zetamarkets/sdk": "^0.16.8",
```
Comment (filipzeta) [inline zeta-cpi/package.json:5]: 0.16.14 is latest, shouldn't have any breaking changes so can just change this number

File: zeta-cpi/tests/zeta_cpi.ts [hunk 2]
```diff
@@ -31,7 +33,9 @@ describe("zeta-cpi", () => {
     throw new Error("ANCHOR_PROVIDER_URL is not defined");
   }
   const connection = new anchor.web3.Connection(url, utils.defaultCommitment());
-  const provider = new anchor.Provider(
+  const asset = assets.Asset.BTC;
```
Comment (filipzeta) [inline zeta-cpi/tests/zeta_cpi.ts:36]: Is this intentionally restricted to just BTC?

Comment (filipzeta) [reply zeta-cpi/tests/zeta_cpi.ts:36]: I think the only issue here is that `pythOracle` gets set to the BTC oracle but should be grabbed when making each transaction

**Review by xjcaa — APPROVED** [review]

**Review by 0xtristan — APPROVED** [review]

File: zeta-cpi/tests/zeta_cpi.ts (hunk 2 above)
Comment (filipzeta) [reply zeta-cpi/tests/zeta_cpi.ts:36]: nvm this is a test pls ignore

**Pull request merged**